### PR TITLE
Added Support For Infinite Recharge

### PR DESCRIPTION
**Pull request by wm-c**
I've added support for the 2020 Field Infinite Recharge

### CODE REVIEW

**Comment by gftabor** [comment]
@carbotaniuman I think this should work but it doesnt seem to for me. Am I missing something

**Comment by carbotaniuman** [comment]
You need to update [this file](https://github.com/wm-c/PathWeaver/blob/master/src/main/java/edu/wpi/first/pathweaver/Game.java) @wm-c. You need to add a static for the game and call `loadGameFromResource()`

**Comment by wm-c** [comment]
I made the change and it should work now.

File: src/main/java/edu/wpi/first/pathweaver/Game.java [hunk 1]
```diff
@@ -13,6 +13,7 @@
 
 public final class Game {
   private static final Set<Game> GAMES = new LinkedHashSet<>();
+  public static final Game INFINTE_RECHARGE = loadGameFromResource("2020-infiniterecharge.json");
```
Comment (Ewpratten) [inline src/main/java/edu/wpi/first/pathweaver/Game.java]: Just for the sake of consistency, this should probably be named `INFINITE_RECHARGE_2020`

**Comment by Ewpratten** [comment]
Line [47](https://github.com/wpilibsuite/PathWeaver/blob/eff54f5261ee9a8002e7d8fd182e66253997a1cc/src/main/java/edu/wpi/first/pathweaver/ProjectPreferences.java#L47) of ProjectPreferences.java should probably also be updated to use the new field as the default

**Comment by gftabor** [comment]
@wm-c works fine now. Where are you getting the image, just want to make sure we use the right image before merging.
Is this figure 4-1 from the manual?

https://firstfrc.blob.core.windows.net/frc2020/Manual/Sections/Section04.pdf

**Comment by wm-c** [comment]
@gftabor I got this image from figure 4-1 from the Infinite Recharge game manual.

**Pull request merged**